### DISCUSSION TITLE
0.107 release on 2021?

### DISCUSSION THREAD

**Discussion by Insight16** [AdguardTeam/AdGuardHome#3917] [Q&A]
Добрый день! Господа разработчики, можно поинтересоваться, а в этом году планируется релиз версии 0.107 или она на неопределенный пириод зависла в бете?)

**Comment by TheCeet** [comment]
Let's hope so...
Release was first planned summer 2021 I guess. It takes a while for releasing 0.107... :(

**Comment by ammnt** [comment]
Me too😪It took so long... I want to get these new functions and bugfixes during holidays!🎅🏼

**Comment by TheCeet** [comment]
Normally there will be one more beta.
See tweet from 11/26/2021:
https://twitter.com/ay_meshkov/status/1464125779424522254

**Comment by ainar-g** [comment]
We hope to publish the first release candidate next week.  If all goes to plan, the actual final release should follow shortly.

**Reply by Insight16** [reply]
Благодарю за ответ! Будем ждать. Спасибо Вам за продукт, очень нравится. Успехов в разработке! :)
